src/lib/components/ListEditor.svelte

```
@@ -1,10 +1,11 @@
 <script lang="ts">
-	import { Button, ComposedModal, ModalHeader, ModalBody, ModalFooter, TextInput } from "carbon-components-svelte"
+	import { Button, ComposedModal, ModalHeader, ModalBody, ModalFooter, TextInput, CodeSnippet } from "carbon-components-svelte"
 
 	import CloseOutline from "carbon-icons-svelte/lib/CloseOutline.svelte"
 	import AddAlt from "carbon-icons-svelte/lib/AddAlt.svelte"
 
 	import { createEventDispatcher } from "svelte"
+	import { Copy } from "carbon-icons-svelte"
```

ESLint

[error] 'Copy' is defined but never used.

(@typescript-eslint/no-unused-vars)